### DISCUSSION TITLE
Chording support

### DISCUSSION THREAD

**Discussion by cflockhart** [espanso/espanso#1543] [Feature requests and Ideas]
Like the CharaChorder X device or ZipChord software, support simultaneous combinations of keys to output text. Not sure if this is possible in Espanso currently.

**Comment by smeech** [comment]
Sorry for the delayed reply, but I have recently started to try to mop up old Issues and Discussions.

You can trigger expansions with CTRL-keys, and sequences thereof, e.g.:
```
  - trigger: "\x05" # <ctrl-e>
    replace: testing
    force_mode: keys
```
but that tends to interfere with many editor menu shortcuts, and Espanso deletes too many characters as the trigger doesn't put one (or more) on-screen. Other key combinations are not currently possible.
